tailwind.config.js

```diff
@@ -42,6 +42,11 @@ export default {
     ],
     theme: {
         extend: {
+            screens: {
+                // Our definition of desktop
+                // Ensure it's the same as defined in screen.ts
+                desktop: '1024px',
```

Comment:
There could be some future work to clarify the intended (max) widths of various pages, some are good at this width, but some use the `lg` breakpoint (1280px)... so there is some inconsistency.

Also, various breakpoints are used for responsiveness, `sm`, `lg`, etc.... would be nice to be clear on intention.